### PR TITLE
10.0 smangukiya 13446 pricelist price

### PR DESCRIPTION
**Pull request by smangukiya**
related task #13346

### CODE REVIEW

File: product_configurator_refactor/models/product_attribute.py [hunk 1]
```diff
@@ -18,9 +18,15 @@ def name_get(self):
...
+                company_currency = self.env.user.company_id.currency_id
+                product_currency = value.product_id.product_tmpl_id.company_id.currency_id
+                val = value.product_id.lst_price
+                if product_currency != company_currency:
+                    val = product_currency.compute(value.product_id.lst_price,
+                                                   company_currency)
                 name = '%s (%s%s)' % (
-                    value.name, currency_id and currency_id.symbol or '$',
+                    value.name,
+                    company_currency and company_currency.symbol or '$',
```
Comment (max3903) [inline product_configurator_refactor/models/product_attribute.py]: @smangukiya The Euro symbol should be after the amount. The currency definition has a field for this:

res_currency

**Review by max3903 — CHANGES_REQUESTED** [review]

File: product_configurator_refactor/models/product_attribute.py (hunk 1 above)
Comment (smangukiya) [reply product_configurator_refactor/models/product_attribute.py]: @max3903 : **Done!**

**Comment by smangukiya** [comment]
@max3903 : Please review again. I have updated.

**Comment by max3903** [comment]
@smangukiya :+1:

**Review by max3903 — APPROVED** [review]

**Pull request merged**